InstantPrintTool.py

```diff
@@ -277,12 +319,51 @@ def __reloadComposers(self, removed=None):
         self.dialogui.comboBox_composers.blockSignals(False)
         if self.dialogui.comboBox_composers.count() > 0:
             self.dialogui.comboBox_composers.setCurrentIndex(active)
-            self.dialogui.spinBoxScale.setEnabled(True)
+            self.dialogui.comboBox_scale.setEnabled(True)
             self.exportButton.setEnabled(True)
         else:
             self.exportButton.setEnabled(False)
-            self.dialogui.spinBoxScale.setEnabled(False)
+            self.dialogui.comboBox_scale.setEnabled(False)
 
     def __help(self):
         manualPath = os.path.join(os.path.dirname(__file__), "help", "documentation.pdf")
         QDesktopServices.openUrl(QUrl.fromLocalFile(manualPath))
+
+    def scaleFromString(self, scaleText):
+        locale = QLocale()
+        parts = [locale.toDouble(part) for part in scaleText.split(":")]
```

Comment:
On second thoughts I don't t think it should be allowed to enter decimal numbers in the scale string. So you can use `locale.toInt(part)`.
Then, you can just add `and not parts[1][0] != 0` in the if conditions below.

Side note: when catching exceptions, only surround the relevant code with the `try` block.